packages/auth/src/signature/middleware.ts

```diff
@@ -80,7 +80,9 @@ function validateSigInputComponents(
   return !(
     !sigInputComponents.includes('@method') ||
     !sigInputComponents.includes('@target-uri') ||
-    (ctx.request.body && !sigInputComponents.includes('content-digest')) ||
+    (ctx.request.body &&
+      Object.keys(ctx.request.body).length > 0 &&
```

Comment:
Afaict, this is is due to `koa-bodyparser`
https://github.com/koajs/bodyparser#usage
> if nothing was parsed, body will be an empty object {}

Would it be worth also enabling [`strict` mode](https://github.com/koajs/bodyparser#options)?
https://github.com/interledger/rafiki/blob/1b03f3e766735ec7c9618bc044838364723ad4f2/packages/auth/src/app.ts#L173

Reply:
Looks like `strict` mode is true by default. So that specifically would be what's causing this behavior, I think.

Reply:
>Default is `true`

Oops.
I wasn't thinking about avoiding this behavior but making sure our new `Object.keys(ctx.request.body).length` is always dealing with an object.